### ISSUE TITLE
Text hyperlink color against light blue background doesn't pass minimum contrast requirements

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#619]
## Describe the task
Hyperlinks against light blue background found in various pages doesn't pass the minimum contrast requirements. We should make this darker.

<details>
<summary>Example Screenshot</summary>

<img width="1172" alt="Screenshot 2024-04-30 at 21 53 47" src="https://github.com/CodeForPhilly/clean-and-green-philly/assets/102739972/9a5fd359-afb2-4b4c-8c38-8207b76d2abf">

</details>

## Acceptance Criteria
- [x] Darken text color of hyperlinks against light blue background found in the following areas:
  - [x] Get Access Page
  - [x] Transform a Property page

## Additional context

* Add any other context about the task here.
* Or here

**Comment by JT0Y** [comment]
I can pull this! 
Do we have any style guideline kind of rules?
I see that the [current colors](https://color.a11y.com/ContrastPair/?bgcolor=E5F8FF&fgcolor=0070f0) are indeed failing.
If there is no specific pattern to follow I can just darken the link colors until it passes to something like this: https://color.a11y.com/ContrastPair/?bgcolor=E5F8FF&fgcolor=1C00F0

**Comment by CodeWritingCow** [comment]
Hi @marvieqa — @JT0Y would like to work on this ticket. Is it ready for development?

**Comment by bacitracin** [comment]
@JT0Y @CodeWritingCow @marvieqa  This is ready for dev, even though we don't have a blue picked out yet to pass the green & light blue backgrounds of the cards that appear on the Transform Property page / Get Access page. 

@JT0Y I think you can keep playing around with blues until you find one that works with the blue/green background, at level AA (4.5:1). When you find it, change the blues for all of the links. It should just be a CSS variable somewhere.

**Comment by bacitracin** [comment]
@JT0Y I assigned you, assuming you'd still like to pick it up. Let me know if not and I'll take you off
